protocol/x/clob/e2e/conditional_orders_test.go

```diff
@@ -20,15 +21,17 @@ import (
 
 func TestConditionalOrder(t *testing.T) {
 	tests := map[string]struct {
-		subaccounts []satypes.Subaccount
-		orders      []clobtypes.Order
+		subaccounts          []satypes.Subaccount
+		orders               []clobtypes.Order
+		ordersForSecondBlock []clobtypes.Order
 
 		priceUpdateForFirstBlock  *prices.MsgUpdateMarketPrices
 		priceUpdateForSecondBlock *prices.MsgUpdateMarketPrices
 
-		expectedExistInState    map[clobtypes.OrderId]bool
-		expectedTriggered       map[clobtypes.OrderId]bool
-		expectedOrderFillAmount map[clobtypes.OrderId]uint64
+		expectedExistInState               map[clobtypes.OrderId]bool
+		expectedInTriggeredStateAfterBlock map[uint32]map[clobtypes.OrderId]bool
+		expectedOrderFillAmount            map[clobtypes.OrderId]uint64
```

Comment:
```suggestion
		expectedOrderFillAmountAfterSecondBlock            map[clobtypes.OrderId]uint64
```

Reply:
actually for these two it's the fourth block. Maybe no need to rename these two fields, but indent them a space different from the other variables and add a comment saying these are checked after all operations in the test